### PR TITLE
Feature: Add serviceInstance Search Support

### PR DESCRIPTION
**Pull request by TinyAllen**
image

1. Fix mock
2. Add serviceInstance Search Support

### CODE REVIEW

**Comment by JaredTan95** [comment]
related : https://github.com/apache/incubator-skywalking/issues/2150

**Comment by JaredTan95** [comment]
image

**Comment by wu-sheng** [comment]
Service instance name have been list, right?

File: .roadhogrc.mock.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ const schema = makeExecutableSchema({ typeDefs: [
   "scalar Long",
   fs.readFileSync('query-protocol/common.graphqls', 'utf8'),
   fs.readFileSync('query-protocol/metadata.graphqls', 'utf8'),
-  fs.readFileSync('query-protocol/database.graphqls', 'utf8'),
+  fs.readFileSync('query-protocol/top-n-records.graphqls', 'utf8'),
```
Comment (wu-sheng) [inline .roadhogrc.mock.js]: Database and top n should be two graphql scripts.

**Comment by JaredTan95** [comment]
> Service instance name have been list, right?

yep.

**Comment by wu-sheng** [comment]
Merged. Please sync main repo.

**Pull request merged**